src/components/experiment-wrapper/experiment-wrapper.js

```diff
@@ -220,7 +339,14 @@ const ExperimentWrapper = ({ theme }) => {
           Oops, this URL isn't valid
         </h2>
         <p className="experiment-wrapper__text">{`${errorMessage}.`}</p>
-        <Button onClick={() => toExperimentTrackingPath()}>Reset view</Button>
+        <Button
+          onClick={() => {
+            toExperimentTrackingPath();
+            history.go();
```

Comment:
This is not an ideal way to do it, I'm still trying to figure out why the page is not loading even when the URL has been updated... I think we have a similar issue in FC too. If you have any idea or suggestion, I'd really appreciate it 😄

Reply:
I've found the issue, it is because the `invalidUrl` is never re-set back to be `false`, hence it keeps stuck on the error message page. I've removed the history.go and redirect it back to `redirectToDefaultRun` where it would check and reset the invalid URL